src/components/VendingMachine/ProductList/ProductList.jsx

```diff
@@ -0,0 +1,16 @@
+import products from "mockData/products";
+
+import ProductItem from "./ProductItem/ProductItem";
+import ProductUl from "./ProductList.styled";
+
+const ProductList = () => {
+  const productsData = products.productsList;
+
+  return (
+    <ProductUl>
+      <ProductItem productsData={productsData} />
+    </ProductUl>
```

Comment:
이렇게 쓰이는 거라면 map을 여기서 사용해서 `ProductItem`을 렌더링하는 게 더 적절할 것 같아요!

```suggestion
    <ProductUl>
      {productsData.map(product => <ProductItem productData={product} />)}
    </ProductUl>
```

Reply:
아래와 같이 변경해 보았습니다! 
```js
const ProductItem = ({ productData }) => {
  return (
    <ProductLi>
      <Button
        data={productData}
        styles={productButtonStyle}
        className="product-button"
      />
      <p className="product-price">{productData.price}</p>
    </ProductLi>
  );
};
```
훨씬 `ProductItem`이라는 이름에 맞고 알기 쉬운 코드가 된 것 같아요! 
감사합니다!🥳